ble_controller/Kconfig

```diff
@@ -19,6 +19,44 @@ config BT_LL_NRFXLIB
 
 endchoice
 
+choice CLOCK_CONTROL_NRF5_IMPL
+        bool
+        prompt "nRF5 Clock controller implementation"
+        default CLOCK_CONTROL_NRF5_NRFXLIB if BT_LL_NRFXLIB
+
+config CLOCK_CONTROL_NRF5_NRFXLIB
+        bool "nRF5 Clock controller from nrfxlib"
+        depends on BT_LL_NRFXLIB
+        help
+          Use Nordic BLE Link Layer implementation.
+
+endchoice
+
+choice CLOCK_CONTROL_NR5_SOURCE
+        prompt "32KHz clock source"
+
+config CLOCK_CONTROL_NRF5_K32SRC_SYNTH
+        bool "Synthesized from HFCLK"
+
+endchoice
+
+choice CLOCK_CONTROL_NR5_ACCURACY
+        prompt "32KHz clock accuracy"
+
+config CLOCK_CONTROL_NRF5_K32SRC_10PPM
```

Comment:
Any reason why not all clock control changes are present here?

Reply:
Rest of them are in clock_control/Kconfig.nrf5